### DISCUSSION TITLE
Shutdown when all transfers are finished option

### DISCUSSION THREAD

**Discussion by yoni-yad2** [iterate-ch/cyberduck#14854] [Ideas]
**Is your feature request related to a problem? Please describe.**
I would like Cyberduck to auto shutdown the machine after all uploads/downloads are over in order to save cloud resource costs.

**Describe the solution you'd like**
A checkbox with label "Shutdown when finished" on the "Transfers" window.

**Describe alternatives you've considered**
Running "shutdown /s /t xxxxx" at windows command line with xxxxx based on Cyberduck time remaining indicator.

**Comment by dkocher** [comment]
My suggestion is to use Cyberduck CLI [^1] to automate these tasks using scripts that block until the transfer is complete.

[^1]: https://duck.sh/

**Reply by yoni-yad2** [reply]
Most download/file managers have this option built in their UI so I think the duck should get with times... 😉
